external_libraries/clipper/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# generate variables with the sources
+set(CLIPPER_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/clipper.cpp)
+
+add_library(clipper STATIC ${CLIPPER_SOURCES})
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+  target_link_libraries(clipper m)
+endif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
```

Comment:
why is this necessary?
Does it works without?

Reply:
That was a copy and paste from triangle. I have removed it for the moment to check if it works.